.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please start by describing concisely the nature of the bug or the feature
+request.
+
+In case of a bug report, please describe the steps how the problem may be
+reproduced. Does the problem occur on production servers? Does it work on the
+development server? Can you reproduce it locally on the `master` branch?
+
+In case of a feature request, please provide the motivation and the concrete use
+case which your request is seeking to address.
+
+Finally, before opening the issue, please select appropriate labels to indicate
+the topic and the complexity of the task. Please select the concrete milestone
+if applicable.
+
+This message should be replaced with the description of the issue at hand.
```

Comment:
Use comments in the template (more info in my main Review):

```markdown
[//]: # "==================
Please provide a description of the issue at hand below.
Thank you.
=========================="
```

This will appear in the issue creation, but won't render even if the user leaves it in.

Reply:
Hmm, I'm not sure about using comments... E.g. some people may think of updating text blob within comment, not paying attention to the arcane syntax around, which could lead to not seeing *any* visible impact, as it were.  My whole template was basically meant as a helper text, just to guide people which things to check first.... and the whole text was meant to be replaced by the submitter.